src/test/java/stepsDefinitions/Hooks.java

```diff
@@ -1,34 +1,39 @@
-package stepsdefinition;
+package stepsDefinitions;
```

Comment:
The package should be named in lower case as convention:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html